ansible/roles/desktop_setup/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Install pycurl
+  apt: name={{item}}
+  with_items:
+    - python-pycurl
```

Comment:
you might use here simple version of apt module call without loop "apt: name=python-pycurl state=present"